src/assets/src/validation.ts

```diff
@@ -66,7 +66,8 @@ export const queueLocationSchema = string().defined().trim().max(100, createRema
 export const uniqnameSchema = string().defined().trim().lowercase()
     .min(3, 'Uniqnames must be at least 3 characters long.')
     .max(8, 'Uniqnames must be at most 8 characters long.')
```

Comment:
The sequence matters for those checks. I think we may need to move the uniqname/not-email check first, the alphabetical check next, and put the min-max length check towards the end.

Now if I enter "zqian$@test.edu", the error message I got is "Uniqnames must be at most 8 characters long."

Reply:
I have reordered it to fix the ordering. Now the email will be triggered first, and the alphabetical check next, then min-max length towards end. 
<img width="645" alt="Screenshot 2024-04-18 at 10 02 15 PM" src="https://github.com/tl-its-umich-edu/remote-office-hours-queue/assets/56746786/3af040b1-e227-4fe3-8f97-2f4155469c78">

Reply:
Tested. The desired check sequence is in place.